scrapers_next/or/committee.py

```diff
@@ -0,0 +1,61 @@
+from spatula import XPath, URL, HtmlListPage, HtmlPage, CSS
+from openstates.models import ScrapeCommittee
+import re
+
+class CommitteeDetail(HtmlListPage):
+    source = URL("https://olis.oregonlegislature.gov/liz/2023R1/Committees/JCT/CommitteeMembers")
+    selector = CSS('tbody tr')
+
+    def process_item(self, item):
+        com = self.input
+        position, name = item.getchildren()
+        name = re.sub(r"Senator |Representative ", "", name.text_content().strip())
+        position = position.text_content().strip()
+        com.add_member(name, position)
+
+        return com
+
+
+class CommitteeList(HtmlListPage):
+    source = "https://olis.oregonlegislature.gov/liz/Committees/list/"
+
+    def process_item(self, item):
+        link = item.get("href").split('/')
+        link = 'https://olis.oregonlegislature.gov/liz/' + link[4] +\
+                "/Committees/" + link[6] + "/CommitteeMembers"
+
+        try:
+            parent = self.parent
+        except:
+            parent = None
+
+        com = ScrapeCommittee(
+                name=item.text_content(),
+                chamber=self.chamber,
+                classification=self.classification,
+                parent=parent,
+            )
+        com.add_source(link)
```

Comment:
Two things here:
1. We try to add a `note` attribute as much as possible here, so we know later in the data where the url is supposed to point to. In this case, you can make it `com.add_source(link, note="Committee Details Page")`.
2. It would be good to add an additional source for the actual list page that you are scraping from in this class. In a spatula page scraper like this you can access the url for the source page as `self.source.url`.
   - so adding that source here would look like `com.add_source(self.source.url, note="Committees List Page")`.

Reply:
Thanks for the inputs, will change this one